app/index.js

```diff
@@ -4,6 +4,13 @@ var chalk = require('chalk');
 var yosay = require('yosay');
```

Comment:
I'm tempted to say we nuke this template and replace it with either the `react` or `webpack` template. Both of those templates generated more concise and clear app scaffolding than this one. Probably something that should be done in a later PR though.
@moxiegirl @yknl thoughts?